drivers/SmartThings/deepsmart/src/commands.lua

```diff
@@ -97,9 +97,18 @@ end
 -- heater mode command
 ----------------
 function command_handler.set_thermostat_mode(driver, device, command)
-  -- just return success as mode is only heat
-  if command.arg.mode == 'heat' then
-    return device:emit_event(caps.thermostatMode.thermostatMode.heat())
+  -- send command
+  local success = wisers.control(device, command, {config.HEATER.ONOFF})
+  -- Check if success
+  if success then
+    if on_off == 'off' then
```

Comment:
where is `on_off` defined?

Reply:
move it to init.lua and delete ip_check_state

Reply:
use command.argument.mode